addons/google_calendar/models/calendar_attendee.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+
+from openerp import api, fields, models
+
+
+class CalendarAttendee(models.Model):
+    _inherit = 'calendar.attendee'
+
+    google_internal_event_id = fields.Char('Google Calendar Event Id')
+    oe_synchro_date = fields.Datetime('Odoo Synchro Date')
+
+    _sql_constraints = [('google_id_uniq', 'unique(google_internal_event_id,partner_id,event_id)', 'Google ID should be unique!')]
+
+    @api.multi
+    def write(self, vals):
+        for record in self:
+            ref = vals.get('event_id', record.event_id.id)
+
+            # If attendees are updated, we need to specify that next synchro need an action
+            # Except if it come from an update_from_google
+            if not self.env.context.get('curr_attendee', False) and not self.env.context.get('NewMeeting', False):
+                self.env['calendar.event'].browse(ref).write({'oe_update_date': datetime.now()})
```

Comment:
If you need current date then Odoo have option then why use python ??
Do it like : 
```
fields.Date.from_string(fields.Date.today())
```